### PR TITLE
ci: Remove quotes from cache key

### PR DESCRIPTION
**Pull request by max-sixty**

### CODE REVIEW

File: .github/set_version.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # since the caches seem to accumulate cruft over time;
 # ref https://github.com/PRQL/prql/pull/2407
 
-grep '^version =' Cargo.toml | tr -d ' ' >> $GITHUB_ENV
+grep '^version =' Cargo.toml | cut -d\" -f2
```
Comment (max-sixty) [inline .github/set_version.sh]:
```suggestion
grep '^version =' Cargo.toml | cut -d\" -f2  >> $GITHUB_ENV
```

**Comment by max-sixty** [comment]
No luck with this — though works locally — is there another way we can remove the quotes @not-my-profile ?

**Comment by not-my-profile** [comment]
How did you observe that the quotes are in the cache key? They actually shouldn't be since in `foo="bar"` the quotation marks are part of the shell syntax.

**Comment by not-my-profile** [comment]
Ah, nvm, I see it.

**Comment by max-sixty** [comment]
The caches are at https://github.com/PRQL/prql/actions/caches

**Comment by not-my-profile** [comment]
Closed in favor of #3122.

**Pull request closed without merging**